docs/contracts/contracts/LSP0ERC725Account/LSP0ERC725Account.md

```diff
@@ -0,0 +1,1485 @@
+**Requirements:**
+
+- If a `value` is provided, the contract MUST have at least this amount in its balance to execute successfully.
+- If the operation type is `STATICCALL` or `DELEGATECALL`, `value` SHOULD be 0.
+- `target` SHOULD be `address(0)` when deploying a contract.
+- MUST pass when called by the owner or by an authorised address that pass the verification check performed on the owner accordinng to [LSP-20-CallVerification] specification.
+
+</blockquote>
+
+<blockquote>
+
+**Emitted events:**
+
+- [`Executed`](#executed) event, when a call is executed under `operationType` 0, 3 and 4
+- [`ContractCreated`](#contractcreated) event, when a contract is created under `operationType` 1 and 2
+- [`ValueReceived`](#valuereceived) event when receiving native tokens.
+
+</blockquote>
+
+#### Parameters
+
+| Name            |   Type    | Description                                                                                           |
+| --------------- | :-------: | ----------------------------------------------------------------------------------------------------- |
+| `operationType` | `uint256` | The operation type used: CALL = 0; CREATE = 1; CREATE2 = 2; STATICCALL = 3; DELEGATECALL = 4          |
+| `target`        | `address` | The address of the EOA or smart contract. (unused if a contract is created via operation type 1 or 2) |
+| `value`         | `uint256` | The amount of native tokens to transfer (in Wei)                                                      |
+| `data`          |  `bytes`  | The call data, or the creation bytecode of the contract to deploy if `operationType` is `1` or `2`.   |
+
+#### Returns
+
+| Name |  Type   | Description |
+| ---- | :-----: | ----------- |
+| `0`  | `bytes` | -           |
+
+### executeBatch
+
+:::note Links
+
+- Specification details in [**LSP-0-ERC725Account**](https://github.com/lukso-network/lips/tree/main/LSPs/LSP-0-ERC725Account.md#executebatch)
+- Solidity implementation in [**LSP0ERC725Account**](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP0ERC725Account/LSP0ERC725Account.sol)
+- Function signature: `executeBatch(uint256[],address[],uint256[],bytes[])`
+- Function selector: `0x31858452`
+
+:::
+
+```solidity
+function executeBatch(
+  uint256[] operationsType,
+  address[] targets,
+  uint256[] values,
+  bytes[] datas
+) external payable returns (bytes[]);
+```
+
+_Calling multiple addresses `targets` using `operationsType`, transferring `values` wei and data: `datas`. _
+
+Batch executor function that behaves the same as [`execute`](#execute) but allowing multiple operations in the same transaction.
+
+<blockquote>
+
+**Requirements:**
+
+- The length of the parameters provided MUST be equal
+- If a `value` is provided, the contract MUST have at least this amount in its balance to execute successfully.
+- If the operation type is `STATICCALL` or `DELEGATECALL`, `value` SHOULD be 0.
+- `target` SHOULD be `address(0)` when deploying a contract.
```

Comment:
I guess we have to say "must" here because it's a requirement, otherwise it reverts

```suggestion
- `target` must be `address(0)` when deploying a contract.
```